src/pages/Lending/AvailablePools.tsx

```diff
@@ -62,6 +64,24 @@ const LendingPoolButton = ({ asset, onPoolClick }: LendingPoolButtonProps) => {
     [isLendingPositionDataLoading, isPoolDataLoading],
   )
 
+  const StatusTag = useCallback(() => {
+    if (poolData?.status === 'Available') {
+      return (
+        <Tag colorScheme='green'>
+          <TagLeftIcon as={CheckCircleIcon} />
+          {translate('common.active')}
+        </Tag>
+      )
+    }
+
+    return (
```

Comment:
Do we want to be exhaustive here, i.e handled `Staged` too?

Reply:
I don't think so, handled `Full` btw, in [8adb4bc](https://github.com/shapeshift/web/pull/7191/commits/8adb4bc5d03daab4737af445ebe41feedd9d88de)